os/include/debug.h

```diff
@@ -501,7 +505,7 @@ int get_errno(void);
 
 #if defined(CONFIG_BUILD_FLAT) || defined(__KERNEL__)
 #define mfdbg(format, ...) if (abort_mode || up_interrupt_context()) \
-                                                             lldbg(format, ##__VA_ARGS__); \
+                                                             assertdbg(format, ##__VA_ARGS__); \
```

Comment:
The definition of `assertdbg` is line 771. It's later of usages. Is it ok?

Reply:
Thank you, I checked it, and I modified to CHECK_SECURE_PERMISSION() macro